addons/website/static/src/snippets/s_image_gallery/000.xml

```diff
@@ -14,7 +14,7 @@
             <div class="carousel-inner" style="padding: 0;">
                  <t t-foreach="images" t-as="image" t-key="image_index">
                     <div t-attf-class="carousel-item #{image_index == index and 'active' or None}">
```

Comment:
Note: `None` should probably be replaced by `''` to avoid having `undefined` as a class. This will be done in another PR as it could be fixed in stable as well.